docs/install.rst

```diff
@@ -137,6 +137,14 @@ environment variable ``SWITCH_THE_DUDE``. If the value of that variable is any o
 is set to one of those "true" values, then all switches will be considered "on" unless they are
 explicitly "off" in the environment.
 
+You may also use these switches in Python in ``views.py`` files. For example::
+
+    from bedrock.base import waffle
```

Comment:
As we just use `{% if switch('dude-and-walter') %}` in the templates, for consistency we could do from `from bedrock.base.waffle import switch` here in the example?